app/frontend/components/pages/login/hardwareAuth.js

```diff
@@ -1,6 +1,9 @@
 const {h} = require('preact')
 const {CRYPTO_PROVIDER_TYPES} = require('../../../wallet/constants')
 
+const {TrezorLogoWhite} = require('../../common/svg')
+const {LedgerLogoWhite} = require('../../common/svg')
```

Comment:
This can be shortened: const {TrezorLogoWhite, LedgerLogoWhite} = require('../../common/svg')

Reply:
Fair enough.